nemo/collections/common/tokenizers/column_code.py

```diff
@@ -0,0 +1,121 @@
+from typing import Dict, List, Tuple, Union
+
+import numpy as np
+from code_spec import Code
+from datatype_code import CategoryCode, FloatCode, IntCode
+
+# from vector_tokenizer import VectorCode
+
+
+column_map = {"int": IntCode, "float": FloatCode, "category": CategoryCode}  # , 'vector': VectorCode}
+
+
+class ColumnCodes(object):
```

Comment:
inherit  from the spec?

Reply:
In a way, the ColumnCodes class is composite, taking in multiple different types of "codes" be it FloatCode, IntCode, or CategoryCode, or potentially something else.